Naveen-gali/Yelp-App
# StoryBook Integration

### What does this PR do?

- Adds Storybook to yelp project

### Related Issues

- closes #26 

### How to test

* Clone the Repository.
* Switch to `dev-StoryBookIntegration` branch.
* Do `yarn` to install the dependencies.
* Do `yarn run build-stage-ios` for `ios` or `yarn run build-stage-android` for `android`.
* If You Dont see story board like in the following screenshots. Go to `.env.staging` and change `SHOW_STORYBOOK ` to **true**.
* Re run the app using `yarn run build-stage-ios` for `ios` or `yarn run build-stage-android` for `android`.
* You Can Notice StoryBook on the screen now.

### Screenshots

#### Android Screen Shots
Screen Shot 1            |  Screen Shot 2| Screen Shot 3 |
:-------------------------:|:-------------------------:|:-------------------------:
![Screenshot_1680691211](https://user-images.githubusercontent.com/122969303/230059338-9b55eb44-26f8-4dfb-a2d7-ef4d75990061.png) | ![Screenshot_1680691205](https://user-images.githubusercontent.com/122969303/230059328-7ac37f02-39b0-4696-a020-36bf561907c8.png) | ![Simulator Screen Shot - iPhone 14 - 2023-04-06 at 15 11 50](https://user-images.githubusercontent.com/122969303/230339572-e78ecc21-4713-47c4-8377-47742e4cf770.png)

#### Ios Screen Shots
Screen Shot 1            |  Screen Shot 2
:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone SE (3rd generation) - 2023-04-05 at 16 01 04](https://user-images.githubusercontent.com/122969303/230059869-b8e875d5-53b1-47aa-afff-bc830caa2012.png) | ![Simulator Screen Shot - iPhone SE (3rd generation) - 2023-04-05 at 16 01 00](https://user-images.githubusercontent.com/122969303/230059884-00b6524f-e07b-4d75-b1a8-4ec0e5358c0d.png)

### Checklist

### Additional Notes


### Packages Added (if any)
| Package | Reason |
| ------- | ------ |
| StoryBook and Its Dependencies        | Packages and Internal Dependencies necessary for Running Storyboard       |
